Ports/bash/patches/0002-We-currently-fail-to-build-getconf.c-because-we-are-.patch

```diff
@@ -0,0 +1,24 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Fabian Dellwing <fabian.dellwing@gmail.com>
+Date: Mon, 24 Apr 2023 18:54:29 +0200
+Subject: [PATCH] We currently fail to build getconf.c because we are missing
+ libintl.h and don't support multiple needed syscalls (from around 300 total
+ syscalls).
+
+---
```

Comment:
This is a very long commit title. It would be better fit for the commit message, and something shorter (like "Remove getopt from examples") for the title.

Reply:
Put a shorter title before and kept the longer explanaition for why we do it.